README.md

```diff
@@ -1,80 +1,140 @@
 ![DeFiScan Logo](./public/images/defiscan_by_dc_color_for_light_background.svg)
 
-A public website for the monitoring of the maturity and risks of DeFi technology.
+# DeFiScan
 
-Read more about the methodology behind the DeFi stages and risks in this [post](https://deficollective.org/blog/introducing-defiscan).
+[![Build](https://github.com/deficollective/defiscan/actions/workflows/nextbuild.yml/badge.svg)](https://github.com/deficollective/defiscan/actions/workflows/nextbuild.yml)
```

Comment:
Would be good to keep the short description here :) 

A public website for the monitoring of the maturity and risks of DeFi technology.